src/tests/Interop/COM/ComWrappers/GlobalInstance/GlobalInstance.TrackerSupport.cs

```diff
@@ -31,15 +31,16 @@ static int Main(string[] doNotUse)
                 // The first test registers a global ComWrappers instance for tracker support.
                 // Subsequents tests assume the global instance has already been registered.
                 ValidateRegisterForTrackerSupport();
-
+#if Windows
```

Comment:
This is the product work that we will be missing and should have a tracking issue for.